src/app/services/utils.service.ts

```diff
@@ -60,12 +60,13 @@ export class UtilsService {
     console.error(missatge);
     this.snackBar
       .open(missatge, 'Cerrar', {
+        duration: 0,
```

Comment:
@albertobucesGdtl this line is a slip during merge; in dev is fixed 

https://github.com/sitmun/sitmun-admin-app/blob/90bf533db43a66256d78aee747fc02ef56ff18fe/src/app/services/utils.service.ts#L58-L61